test/Base_Tests/src/Runtime/GC_Example.enso

```diff
@@ -15,24 +15,31 @@ type My_Resource
 
 close_resource resource = resource.close
 
-repeat_cleanup_until_done counter println =
+repeat_cleanup_until_done holder counter println =
     go i =
         if counter.get == 0 then Nothing else
+            if i % 100 == 3 then
+                println "Releasing holder with resources: "+holder.get.length.to_text
+                holder.put []
```

Comment:
This will mean that this message is printed every 100 iterations. But it suffices to do this only once. Can we remove the `% 100`?

```suggestion
            if i == 3 then
                println "Releasing holder with resources: "+holder.get.length.to_text
                holder.put []
```